Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationBroadcaster.cs

```diff
@@ -40,6 +52,11 @@ public class StateSynchronizationBroadcaster : Singleton<StateSynchronizationBro
         private const float PerfUpdateTimeSeconds = 1.0f;
         private float timeUntilNextPerfUpdate = PerfUpdateTimeSeconds;
 
+        /// <summary>
+        /// The Localization method to be used by the observer.
```

Comment:
>Localization [](start = 16, length = 12)

Is this a term commonly used for this? When I hear "localization" I immediately think of the process of translating an application's text for multiple languages. Maybe calling it SpatialLocalization would help make that more clear?